test-suite/unit-tests/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "",
   "scripts": {
-    "build": "glob-exec --foreach \"contracts/*.gr\" -- \"node script/grainfind.js compile {{file}}\"",
+    "build": "glob-exec --foreach \"contracts/*.gr\" -- \"grain compile -I ../../ --wasi-polyfill ../../wasi.gr --no-bulk-memory --use-start-section --no-gc --elide-type-info --release {{file}}\"",
```

Comment:
Do we also want to add a .vscode/settings.json file that sets these flags in the LSP?

Reply:
done